config/software/ruby-cleanup.rb

```diff
@@ -146,51 +156,62 @@
       CHANGES.md
       CHANGES.txt
       CODE_OF_CONDUCT.md
+      Code-of-Conduct.md
+      codecov.yml
+      concourse
       CONTRIBUTING.md
       CONTRIBUTING.rdoc
       CONTRIBUTORS.md
-      Code-of-Conduct.md
+      doc
+      doc-api
+      docker-compose.yml
+      Dockerfile*
+      docs
+      donate.png
+      ed25519.png
```

Comment:
I _guess_ I can live without a diagram of how ed25519 works.

![eye roll](https://user-images.githubusercontent.com/517302/88215124-300b6500-cc29-11ea-9826-2e93b1cb15da.gif)